### DISCUSSION TITLE
Reusable Blocks

### DISCUSSION THREAD

**Discussion by denizs** [GitbookIO/community#628] [Feature Requests]
Hey dear Gitbook community, I love what you are doing here ❤️ 

I'm currently using Gitbook to write technical documentation and am finding myself copy-pasting the same blocks with the same content on multiple pages. Specifically, it's a table containing a set of IPs that might change in the future and if so, these changes would need to be applied on every table. Maybe it's just me, but after doing copy-pasta for the third time, I felt the urge to somehow centralize this block so that changes do not have to be applied n times.

Hence, my idea would be to enable users to create custom blocks (with predefined content), that they could then use on any page, where a change to the content of the block is reflected everywhere. I haven't thought much about the UX (i.e. do you need a dedicated edit for the master or do you simply propagate changes from any table to the master), but I'm sure you'll know what to do best :)

**Comment by aikoii** [comment]
@denizs Thanks for you kind words 💙! Excited to let you know that we're already working on a Synced Blocks feature, You can read more about it here: https://github.com/orgs/GitbookIO/discussions/121#discussioncomment-9444957

**Reply by denizs** [reply]
@aikoii thats great to hear! Thanks a bunch for the reply! :-)

**Comment by derloos** [comment]
Yes, please! Great suggestion! Like, if you need to make the same compatibility issue warning across multiple pages, etc.

**Comment by SamyPesse** [comment]
It's shipping today: https://github.com/orgs/GitbookIO/discussions/772 🎉
